GOVERNANCE.md

```diff
@@ -217,6 +257,27 @@ push commits, etc.), so what's the minimal amount is subjective, and there will
 be cases where collaborators disagree on whether a nomination should move
 forward.
 
+Refrain from discussing or debating aspects of the nomination process
+itself directly within a nomination private discussion or public issue.
+Such discussions can derail and frustrate the nomination and cause unnecessary
+friction. Move such discussions to a separate issue or discussion thread.
+
+##### How to oppose a collaborator nomination
+
+Explicit opposition would typically be signaled as some form of clear
+and unambiguous comment like, "I don't believe this nomination should pass".
```

Comment:
IMO it's worth point out that opposition doesn't need to be public, and can be anonymous.
```suggestion
Explicit opposition would typically be signaled as some form of clear
and unambiguous comment like, "I don't believe this nomination should pass".
It doesn't need to be done publicly, but the TSC is more likely to dismiss an
opposition done anonymously. Ideally, the comment showing opposition should
be done in the private discussion, ideally before the public issue is opened.
```

We should probably change the bullet point on line 245 "If you think the nomination was made too soon, …" as it repeats a lot of the content of this section, it should instead refer to it.